src/main/java/Duke.java

```diff
@@ -1,10 +1,93 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
+        String lineBreak = "---------------------------------------------";
```

Comment:
Coding Standard: Since this is a constant, you have to capitalise the full name.

```suggestion
        String LINEBREAK = "---------------------------------------------";
```